at_end2end_test/pubspec.yaml

```diff
@@ -15,7 +15,12 @@ dependencies:
   at_lookup: ^3.0.22
   at_client: ^3.0.20
 
-
+dependency_overrides:
+  at_commons:
+    git:
+      url: https://github.com/atsign-foundation/at_tools.git
+      path: at_commons
+      ref: exception_handling_changes
```

Comment:
Please make a note at the top of your PR noting the dependency override and the need, once the new at_commons is available, to remove this dependency override.

Reply:
Added a Note on why dependency-overrides is un-commented.